src/Parsers/ASTCreateQuery.cpp

```diff
@@ -240,6 +240,8 @@ ASTPtr ASTCreateQuery::clone() const
 
     if (columns_list)
         res->set(res->columns_list, columns_list->clone());
+    if (aliases_list)
```

Comment:
Don't we need to change something in `ASTCreateQuery::formatImpl` and `ASTCreateQuery::formatQueryImpl` to print the aliases if the formatted query?